### PR TITLE
adds --scale range option

### LINKED ISSUE #822: support --scale
I find myself often needing to do: `--min-scale=X --max-scale=x`, usually when x=1.

It might be nice to support a short-cut here: `--scale=x` to mean "set both min & max".

Just a thought...

### PR DESCRIPTION
**Pull request by mpetason**
## Description

Fixes #822

## Changes

* adds --scale range option, #..#

## Reference

Fixes #822 

Replaces previous PR where I had issues with a rebase - #1070 
Replaces previous PR where I had issues with a rebase - #986

### CODE REVIEW

**Comment by mpetason** [comment]
@maximilien

**Comment by rhuss** [comment]
@mpetason Thanks a lot for all your work. Let's get that landed, I will give a review this week so that we finally be able to merge it. Thanks also for your patience.

**Review by maximilien — APPROVED** [review]
LGTM

**Review by maximilien — APPROVED** [review]
/lgtm

**Comment by mpetason** [comment]
It doesn't look like the failing test is associated with my changes. I'll try running it again on Monday.

File: pkg/kn/commands/service/configuration_edit_flags.go [hunk 1]
```diff
@@ -264,15 +268,41 @@ func (p *ConfigurationEditFlags) Apply(
...
 			if err != nil {
 				return err
 			}
-			err = servinglib.UpdateMinScale(template, p.Scale)
+			err = servinglib.UpdateMinScale(template, scaleMin)
 			if err != nil {
 				return err
 			}
+		} else {
+			return fmt.Errorf("Scale must be of the format x..y or x")
```
Comment (navidshaikh) [inline pkg/kn/commands/service/configuration_edit_flags.go:305]:
```suggestion
			return fmt.Errorf("scale must be of the format x..y or x")
```

Comment (navidshaikh) [reply pkg/kn/commands/service/configuration_edit_flags.go:305]: error messages should begin with lower case

Comment (mpetason) [reply pkg/kn/commands/service/configuration_edit_flags.go:305]: Should I make this edit and create a new PR, or commit suggestion? Not sure how it works after a merge.

**Pull request merged**